mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -123,7 +123,7 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	METHOD method_1508 handleInputEvents ()V
 	METHOD method_1511 doItemPick ()V
 	METHOD method_1513 reloadResourcesConcurrently ()Ljava/util/concurrent/CompletableFuture;
-	METHOD method_1514 start ()V
+	METHOD method_1514 run ()V
```

Comment:
Does it clash with `java.lang.Runnable#run()V`?

Reply:
No, `MinecraftClient` doesn't implement runnable.